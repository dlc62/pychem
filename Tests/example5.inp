[example5]
Method = "HF"
Basis_Sets = ["STO-3G"]
Multiplicity = 1
Charge = 0
Coords_Units = "BOHR"
Coords = [['H', 1.0,  0.  ,  0.,   0.],
          ['H', 1.0, 1.398397232,   0.,   0.]]
Reference = "RHF"
Max_SCF_Iterations = 50
Job_Type = "PROPERTY"
Property_Type = "SCATTERING"
Property_Grid = [0.0]
#Property_Grid = [0.0,0.5,1.0,1.5,2.0,2.5,3.0,3.5,4.0,4.5,5.0]

