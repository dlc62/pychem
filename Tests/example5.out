-------------------------------------------------------
 Hartree-Fock iterations 
-------------------------------------------------------

 Cycle:  1
 Total energy:  -1.11675932391
 Change in energy:  0.674755939362

 Cycle:  2
 Total energy:  -1.11675932391
 Change in energy:  0.0
 DIIS error:  6.68860165862e-17

-------------------------------------------------------
 End of Hartree-Fock iterations 
-------------------------------------------------------

Final HF energy:  -1.11675932391


 Alpha Orbital Energies 
[-0.57855389  0.67114356]

 Alpha MOs 
[[ 0.54884227 -1.21245199]
 [ 0.54884227  1.21245199]]

 Alpha Density Matrix 
[[ 0.30122784  0.30122784]
 [ 0.30122784  0.30122784]]


-------------------------------------------------------
 End of Hartree-Fock calculation 
-------------------------------------------------------

-------------------------------------------------------
 Property calculation - electron scattering intensities 
-------------------------------------------------------

Grid value -> Scattering patterns for each electronic state 

  0.000000  4.000000000000

-------------------------------------------------------
 End of property calculation 
-------------------------------------------------------

